CaloConditions/inc/CalCalib.hh

```diff
@@ -30,10 +30,13 @@ namespace mu2e {
       const CalCalibPar& calib(std::uint16_t roid) const {
         return _cvec.at(roid);
       }
+      CalCalibPar const* calCalib() const { return _calpar.get(); }
+      void print( std::ostream& ) const;
 
   private:
 
       CalibVec _cvec;
+      std::unique_ptr<CalCalibPar> _calpar;
```

Comment:
I don't understand this.  I expect the entity to hold a CalCalibPar for each channel which is there as _cvec, so why is this lone one here?  The print dump should loop over _cvec.

Reply:
oops, I was trying to figure something out but should remove now